addons/account/tests/test_sequence_mixin.py

```diff
@@ -395,6 +395,25 @@ def test_sequence_concurency(self):
             account.unlink()
             env0.cr.commit()
 
+    def test_resequence_clash(self):
+        """Resequence doesn't clash when it uses a name set in the same batch
+        but that will be overriden later."""
+        moves = self.env['account.move']
+        for i in range(3):
+            moves += self.create_move(name=str(i))
+        moves.action_post()
+
+        mistake = moves[1]
+        mistake.button_draft()
+        mistake.posted_before = False
+        mistake.unlink()
+        moves -= mistake
+
+        self.env['account.resequence.wizard'].create({
+            'move_ids': moves.ids,
+            'first_name': '2',
+        }).resequence()
```

Comment:
Would have been nice to test that the result is also what you expected

Reply:
Tester c'est douter
Mais sinon c'est vrai